bpfman/src/types.rs

```diff
@@ -625,14 +625,14 @@ impl ProgramData {
                         // always use the provided program name.
                         let provided_name = self.get_name()?.clone();
```

Comment:
The above comment isn't correct any more because we require that the user provide a name, and that name is present in the metadata.